ios/Codeforces Watcher/Application/features/users/cells/userAccount/UserAccountViewCell.swift

```diff
@@ -0,0 +1,137 @@
+import SwiftUI
+
+struct UserAccountViewCell: View {
+    
+    var avatar: String = ""
+    
+    var handle: String = ""
+    var name: String = ""
+    var rating: NSMutableAttributedString = "".attributed
+    var maxRating: NSMutableAttributedString = "".attributed
+    var rank: NSMutableAttributedString = "".attributed
+    var contribution: NSMutableAttributedString = "".attributed
+    var dateOfLastRatingUpdate: String = "".localized
+    
+    var onViewProfile: (String) -> Void = { _ in }
+    
+    var rankColor: UIColor {
+        getColorByUserRank(rank.string.lowercased())
+    }
+    
+    var body: some View {
```

Comment:
This light looks quite complicated. Let's try extracting private components (and maybe some public if they can be reused) to simplify the code. For example, you can extract the icon/text component, left and right components, so on.